packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -119,7 +141,7 @@ export async function stripeCheckout({
           upgradeFromPurchase,
       )
 
-      const TWELVE_FOUR_HOURS_FROM_NOW = Math.floor(
+      const TWELVE_HOURS_FROM_NOW = Math.floor(
```

Comment:
I noticed this variable renaming from `TWENTY_FOUR_HOURS_FROM_NOW` to `TWELVE_FOUR_HOURS_FROM_NOW` and fixed it up throughout this file.